package.json

```diff
@@ -18,7 +18,7 @@
     "housekeeping": "npm run update:all && npm run install:all && npm run lint:fix:all && npm run audit:all",
     "postinstall": "husky install",
     "install:all": "npm i --legacy-peer-deps && cd ./generate-exip-pricing-grid && npm i --legacy-peer-deps && cd .. && cd ./src/ui && npm i --legacy-peer-deps && cd .. && cd ./api && npm i --legacy-peer-deps",
-    "lint": "npm i --legacy-peer-deps && npm i --legacy-peer-deps --prefix ./src/ui && eslint --ext .ts src/ui && npm i --legacy-peer-deps --prefix ./src/api && eslint --ext .ts src/api && eslint --ext .js e2e-tests",
+    "lint": "npm i --legacy-peer-deps && cd ./src/ui && npm i --legacy-peer-deps && eslint --ext .ts ./ && cd .. && cd ./api && npm i --legacy-peer-deps && eslint --ext .ts ./ && cd .. && cd .. && cd /e2e-tests && eslint --ext .js ./",
```

Comment:
--prefix flag should work fine.

Reply:
prefix doesnt work locally as it adds a local dependency to the package.json which then when pushed, breaks all tests
https://stackoverflow.com/questions/74276339/how-to-disable-local-package-dependency-for-npm-install-prefix